api/src/pcapi/core/offerers/validation.py

```diff
@@ -74,13 +74,11 @@ def check_venue_edition(modifications: dict[str, typing.Any], venue: models.Venu
     if managing_offerer_id:
         raise ApiErrors(errors={"managingOffererId": ["Vous ne pouvez pas changer la structure d'un lieu"]})
     # modifications.get("siret") may be False if there is no change (ok) OR if it has been cleared (forbidden!)
-    if "siret" in modifications and venue.siret and siret != venue.siret:
-        raise ApiErrors(errors={"siret": ["Vous ne pouvez pas modifier le siret d'un lieu"]})
     if siret:
         venue_with_same_siret = models.Venue.query.filter_by(siret=siret).one_or_none()
         if venue_with_same_siret:
             raise ApiErrors(errors={"siret": ["Un lieu avec le même siret existe déjà"]})
-    if "name" in modifications and modifications["name"] != venue.name:
+    if "name" in modifications and modifications["name"] != venue.name and (siret is None or venue.siret == siret):
```

Comment:
Pourquoi est-ce que ce n'est pas "seulement":
```suggestion
    if "name" in modifications and modifications["name"] != venue.name and venue.siret:
```

Reply:
Car on ne veut permettre la modification de la raison social ssi le siret a changé ou alors si le lieu ne possède pas de `siret` 

Je rate peut être qq chose dans ta proposition mais j'ai l'impression qu'on autoriserait la modification de la raison sociale même si le siret n'a pas changé

Reply:
Je suis d'accord avec ta proposition antoine, un utilisateur ne peut jamais changer directement la raison sociale d'ailleurs non ? 
Pour les lieux avec siret, il viens du siret. Pour les lieux sans siret c'est uniquement à la création ?

Reply:
Oui c'est exactement ça 👍